functies.php

```diff
@@ -15,27 +15,58 @@ function getKabinet(): string
 
 function getBitcoinPrice(): string
 {
-    $url = 'https://api.coinbase.com/v2/exchange-rates?currency=BTC';
+    $url = 'https://api.kucoin.com/api/v1/market/stats?symbol=BTC-EUR';
     $jsonData = file_get_contents($url);
+
+    // Check if the data was fetched correctly
+    if ($jsonData === false) {
+        return 'Error fetching data from kucoin';
```

Comment:
Is een foutmelding in het Nederlands niet netter? Zeker omdat hij direct door de bot wordt getoond.